roles/prometheus-postgres-exporter/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: restart prometheus-postgres-exporter
+  become: true
+  systemd:
+    daemon_reload: true
+    name: prometheus-postgres-exporter.service
+    state: restarted
```

Comment:
Can this instead use the `service` module which is a bit more generic, or is the a good reason to be using `systemd` here?

Reply:
No, I don't believe there is a reason to use systemd here over service.

Reply:
https://github.com/python-discord/infra/pull/58/commits/075e7d8b15646bdf62d3b04f3654b81a4f3b20e9

Reply:
🍣🍣🍣🍣🍣